src/sass/components/_promotions.scss

```diff
@@ -0,0 +1,178 @@
+.wrapper {
+  max-width: 1140px;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  box-sizing: border-box;
+  padding-bottom: 60px;
+  margin: -50px auto 0 auto;
+  padding: 0 15px;
+
+  .main-promotion {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 540px;
+    height: 420px;
+    background-image: url("../images/promotions/living-room.png");
+    background-size: cover;
+
+    .description {
+      position: relative;
+      width: 489px;
+      height: 355px;
```

Comment:
czy ustalanie sztywnych szerokosci i/lub wysokosci jest konieczne? duzo elementów w tym pliku ma to ustawione na sztywno.

nie mozemy tego rozwiązać w inny sposob?

Reply:
Co do wysokości - tak, jest to konieczne ze względu na treść taska, którą przytoczyłem w opisie.
Szerokość ustawiona na sztywność to już mój zabieg, który miał na celu sprawdzić skalowanie zdjęć ( w projekcie szerokość głównych elementów to 555px,  u mnie 545px.

Jakie inne rozwiązanie Twoim zdaniem byłoby lepsze?

Reply:
rozważałbym ewentualnie ustalenie szerokości kontenera a wewnatrz albo `100%` albo jakimis innymi zabiegami typu flexbox

ale ponad tym wszystkim - najwazniejsze ze dziala, wiec jesli to mialoby zajac niewiadomo ile czasu, to nie warto w tym wypadku 👍

Reply:
Na przyszłość na pewno zapamiętam! :)